src/stories/events/kinds/30001.stories.ts

```diff
@@ -8,7 +8,7 @@ import Kind30001 from '../../../lib/event/content/Kind30001.svelte';
  */
 
 const meta = {
-    title: 'Event/Kinds/Lists/30001',
+    title: 'Event/Kinds/30001',
```

Comment:
I like the `<concern>/<kind>` structure when possible; otherwise `Event/Kinds` will be impossible to navigate